### PR TITLE
Renames kube-dns configure files from skydns* to kubedns*

### PR DESCRIPTION
**Pull request by MrHohn**
`skydns-` prefix and `-rc` suffix are confusing and misleading. Renaming it to `kubedns` in existing yaml files and scripts.

@bowei @thockin

### CODE REVIEW

File: cluster/ubuntu/deployAddons.sh [hunk 1]
```diff
@@ -41,15 +41,15 @@ function init {
 
 function deploy_dns {
   echo "Deploying DNS on Kubernetes"
-  sed -e "s/\\\$DNS_DOMAIN/${DNS_DOMAIN}/g" "${KUBE_ROOT}/cluster/addons/dns/skydns-rc.yaml.sed" > skydns-rc.yaml
```
Comment (thockin) [inline cluster/ubuntu/deployAddons.sh]: We should also fix the sed file to not use `$DNS_DOMAIN` but `VAR_DNS_DOMAIN` or `__DNS_DOMAIN` or something, so we don't have to double-escape like this...

**Comment by thockin** [comment]
LGTM

**Comment by deads2k** [comment]
This broke `local-up-cluster.sh`. @xilabao this may have been your problem.

**Comment by MrHohn** [comment]
@deads2k I should be more careful. Thanks for fixing the scripts!

**Pull request merged**